### PR TITLE
Design/show color status

### LINKED ISSUE #103: Feature Request: Make STATUS of printers more "visible"
Either (or both) making the background of the title cell or the STATUS cell to different colors (user selectable on settings?) depending on what's happening so we can have a quick look at the farm status when we have many printers to take care of.

A quick photoshop mockup of both options (I think both is more visual):

mokcup

### PR DESCRIPTION
**Pull request by MaximeBerduck**
Closes #103 

Easy way to display the printer status in colour. 
- PRIMARY (blue) -> isPrinting
- ERROR (red) -> isError (no change)
- SUCCESS (green) -> isReady

image

### CODE REVIEW

File: bambu/src/main/java/com/tfyre/bambu/view/dashboard/DashboardPrinter.java [hunk 1]
```diff
@@ -318,9 +318,21 @@ private void updatePrinterStatus() {
...
+            printerStatus.removeClassName(LumoUtility.TextColor.PRIMARY);
+            printerStatus.addClassName(LumoUtility.TextColor.SUCCESS);
+        } else if(gcodeState.isPrinting()) {
+            printerStatus.removeClassName(LumoUtility.TextColor.ERROR);
+            printerStatus.removeClassName(LumoUtility.TextColor.SUCCESS);
+            printerStatus.addClassName(LumoUtility.TextColor.PRIMARY);
         } else {
             printerStatus.removeClassName(LumoUtility.TextColor.ERROR);
+            printerStatus.removeClassName(LumoUtility.TextColor.SUCCESS);
+            printerStatus.removeClassName(LumoUtility.TextColor.PRIMARY);
```
Comment (TFyre) [inline bambu/src/main/java/com/tfyre/bambu/view/dashboard/DashboardPrinter.java:335]:
```suggestion
        printerStatus.removeClassName(LumoUtility.TextColor.ERROR);
        printerStatus.removeClassName(LumoUtility.TextColor.PRIMARY);
        printerStatus.removeClassName(LumoUtility.TextColor.SUCCESS);
        if (gcodeState.isError()) {
            printerStatus.addClassName(LumoUtility.TextColor.ERROR);
        } else if(gcodeState.isReady()){
            printerStatus.addClassName(LumoUtility.TextColor.SUCCESS);
        } else if(gcodeState.isPrinting()) {
            printerStatus.addClassName(LumoUtility.TextColor.PRIMARY);
        }
```

**Comment by TFyre** [comment]
If you are happy with the suggestion, I'll squash & merge ;)

**Comment by MaximeBerduck** [comment]
Yes it will better with your suggestion ! 🙂

**Comment by TFyre** [comment]
Thank you for the contribution!

**Comment by bolsoncerrado** [comment]
Thank you guys.

**Pull request merged**